src/landscape/fow/C4FoWAmbient.h

```diff
@@ -49,11 +49,20 @@ class C4FoWAmbient
 	unsigned int SizeY{0};
 	// Brightness (not premultiplied)
 	double Brightness{1.};
+	long Color = 0xffffffff; // Return value for GetColor and object script
```

Comment:
IMO this should be an `uint32_t`. I know the script interface uses `long` everywhere, but really we want a type that can store four independent bytes and not a signed number that has a size of maybe 4 or 8.

Reply:
Hmm, ok